documentation/book/ref-storage.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// assembly-storage.adoc
+
+[id='ref-storage-{context}']
+= Storage
+
+Storage can be configured using the `storage` property in following resources:
+
+* `Kafka.spec.kafka`
+* `Kafka.spec.zookeeper`
+
+{ProductName} supports two types of storage:
+
+* Ephemeral
+* Persistent
+
+The type of storage is specified in the `type` field.
+
+== Ephemeral storage
+
+Ephemeral storage is using the {K8sEmptyDir} to store tha data.
+To use ephemeral storage, the `type` field should be set to `ephemeral`.
+
+IMPORTANT: `EmptyDir` volumes are not persistent and the data stored in them will be lost when the Pod is restarted.
+After the new pod is started, it has to recover all data from other nodes of the cluster.
+Ephemeral storage is not suitable for use with single node Zookeeper clusters and for Kafka topics with replication factor 1, because it will lead to data loss.
+
+.An example of Ephemeral storage
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    # ...
+    storage:
+      type: ephemeral
+    # ...
+  zookeeper:
+    # ...
+    storage:
+      type: ephemeral
+    # ...
+----
+
+== Persistent storage
+
+Persistent storage is using {K8sPersistentVolumeClaims} to provisine persistent volumes for storing data.
```

Comment:
- Change  "Persistent storage uses {K8sPersistentVolumeClaims} to provisine persistent" to "Persistent storage uses {K8sPersistentVolumeClaims} to provision persistent " (typo and reduced a word)
- delete many from "provision volumes of many different types"